### ISSUE TITLE
Add/Delete Task Functionality

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#762]
**We have a feature branch `feature/#448-task-lists` for this epic. Make sure to branch from/PR into this branch!**

Must be completed after #760 #761 Get Tasks Endpoint, Create Task Endpoint, and Delete Task Endpoint (replace with ticket references once made).

Make datagrids populate with the actual task data from the backend. Hook up the delete button from the action button on the right. When clicked it should pop up a modal that asks if you're sure you want to delete the task. If the user clicks 'confirm' the task should be deleted, if they click 'cancel', it should go back to the state before the modal was opened. 

Wire up the add task button as shown below. When the add task button is clicked, a new row should appear in the data grid, allowing you to type in what you want for each of the fields, and having save and delete buttons on the left. Assignee should be an [multiple value autocomplete textfield](https://mui.com/material-ui/react-autocomplete/#combo-box)(multiple values) with options of only members of the project’s team, tasks will be able to have multiple assignees The delete button should just cancel the new task and take you back to before you pressed add task, the save button should actually create the new task and add it to the current data grid. 

![image](https://user-images.githubusercontent.com/29521172/215366029-fff27871-32cc-49e0-888c-eddaab4c8541.png)
